ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.js

```diff
@@ -0,0 +1,490 @@
+      durationSeconds: 4,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Paradigm Knave Colossal Impact Sides',
+      netRegex: NetRegexes.startsUsing({ id: '5EA4', source: 'Knave Of Hearts', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5EA4', source: 'Herzbube', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5EA4', source: 'Jack', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5EA4', source: 'ジャック', capture: false }),
+      durationSeconds: 5,
+      response: Responses.goMiddle(),
+    },
+    {
+      id: 'Paradigm Copied Knave Colossal Impact Sides',
+      netRegex: NetRegexes.startsUsing({ id: '5EA4', source: 'Copied Knave', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5EA4', source: 'Kopiert(?:e|er|es|en) Herzbube', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5EA4', source: 'Réplique De Jack', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5EA4', source: '複製サレタジャック', capture: false }),
+      // Cast time of 8 seconds, clones start casting 6 seconds into the cast.
+      delaySeconds: 2.1,
+      durationSeconds: 5,
+      response: Responses.goMiddle(),
+    },
+    {
+      id: 'Paradigm Knave Colossal Impact Middle',
+      netRegex: NetRegexes.startsUsing({ id: '5EA7', source: 'Knave Of Hearts', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5EA7', source: 'Herzbube', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5EA7', source: 'Jack', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5EA7', source: 'ジャック', capture: false }),
+      durationSeconds: 5,
+      response: Responses.goSides(),
+    },
+    {
+      id: 'Paradigm Copied Knave Colossal Impact Middle',
+      netRegex: NetRegexes.startsUsing({ id: '5EA7', source: 'Copied Knave', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5EA7', source: 'Kopiert(?:e|er|es|en) Herzbube', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5EA7', source: 'Réplique De Jack', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5EA7', source: '複製サレタジャック', capture: false }),
+      delaySeconds: 2.1,
+      durationSeconds: 5,
+      response: Responses.goFrontBack(),
+    },
+    {
+      // Also applies for Red Girl Manipulate Energy
+      id: 'Paradigm Knave Magic Artillery Beta You',
+      netRegex: NetRegexes.headMarker({ id: '00DA' }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: 4,
+      response: Responses.tankBuster('alert'),
+    },
+    {
+      id: 'Paradigm Knave Magic Artillery Beta Collect',
+      netRegex: NetRegexes.headMarker({ id: '00DA' }),
+      run: function(data, matches) {
+        data.busterTargets = data.busterTargets || [];
+        data.busterTargets.push(matches.target);
+      },
+    },
+    {
+      id: 'Paradigm Knave Magic Artillery Beta Not You',
```

Comment:
```suggestion
      id: 'Paradigm Knave Magic Artillery Beta',
```

Reply:
Sure! Took this from the other alliance raid